sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwaptionExamples.cs

```diff
@@ -132,5 +133,113 @@ public void InterestRateSwaptionInlineValuationExample(ModelSelection.ModelEnum
             var swaption = InstrumentExamples.CreateExampleInterestRateSwaption();
             CallLusidInlineValuationEndpoint(swaption, model);
         }
+        
+        /// <summary>
+        /// Lifecycle management of swaption
+        /// For both cash and physically settled swaption, we expected conservation
+        /// of PV (under CTVoM model) and in particular, equals to the payoff
+        ///
+        /// Cash-settled case: works the same as the others. This means, we get the cashflow
+        /// and upsert that back into LUSID.
+        ///
+        /// Physically-settled case: Cashflow would be paying the strike to obtain the underlying.
+        /// There is additional code to get the underlying in the GetValuation call as well as then
+        /// upserting the underlying back into the portfolio. 
+        /// </summary>
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney,"Cash")]
```

Comment:
this needs a unique LusidFeature (https://gitlab.com/finbourne/salesengineering/lusid-features/-/blob/master/features.yml#L365) most likely a F22-x

Reply:
@azvyagin1, I can add the above key for you (I'm adding some on my side currently).

Reply:
thank you!

Reply:
It's "F22-35" :)

Reply:
from https://gitlab.com/finbourne/salesengineering/lusid-features/-/commit/651bcad4257edda4e7c23a4f4148a65ff4fbc6ed

Reply:
Need a rebase / resolve conflict btw